package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "@elastic/package-registry",
+  "version": "0.2.0",
```

Comment:
Ideally this would be linked to the `version` in `BuildRootFile` so they're always the same

Reply:
We probably will have to generate it. Or does it allow linking here?

Reply:
What I was imagining was BuildRootFile would read package.json to get the version.

package.json would be managed with [`npm version`](https://docs.npmjs.com/cli/version) command or something similar. 

The root file would remain synced and the build script doesn't have to take on knowledge of building another artifact. 

Version would gain a source of truth that can be read by other tools instead of being inside a script.

Reply:
+1 on having it in a file. I wonder if `package.json` is the right place for it considering it is a Golang tool, but in the end it is just JSON, so ... Will figure it out as soon as we automate it.